modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -96,6 +96,8 @@ spec:
         mode: 420
         overwrite: true
         path: /etc/clevis.json
+  kernelArguments:
+    - rd.neednet=1
```

Comment:
I think it would be beneficial for the reader to add a call out for this kernel argument stating exactly what it is doing. It might also be useful to include a note under the call out, stating that it is required when configuring Tang.

Regarding what it is doing, please see my previous comment.